test/merge-params-test.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const {assert} = require('chai');
+const {mergeParams} = require('../src/metamorphoses');
+
+
+describe('Metamorphoses • Utility', function() {
+  context('Merging 0 resource parameter with 1 action parameter', function() {
+    const actionParams = [{
+      key: 'name',
+      type: 'string',
+      values: [],
+      required: false,
+      example: 'name value',
+      default: null,
+      description: 'Name'
+    }];
+
+    const params = mergeParams([], actionParams);
+
+    it('should give 1 parameter', () => assert.equal(params.length, 1));
+
+    return it('should have action parameter', () => assert.equal(params[0].key, 'name'));
+  });
+
+  context('Merging 1 resource parameter with 0 action parameter', function() {
+    const resourceParams = [{
+      key: 'id',
+      type: 'string',
+      values: [],
+      required: false,
+      example: 'id value',
+      default: null,
+      description: 'Id'
+    }];
+
+    const params = mergeParams(resourceParams);
+
+    it('should give 1 parameter', () => assert.equal(params.length, 1));
+
+    return it('should have resource parameter', () => assert.equal(params[0].key, 'id'));
```

Comment:
no return. Usually a good idea to put the assertion on separate line. Same in above test too.